ecl/hql/hqlexpr.cpp

```diff
@@ -9345,7 +9345,7 @@ IHqlExpression * createDelayedReference(node_operator op, IHqlExpression * modul
             ret.setown(createDelayedScope(ret.getClear()));
     }
 
-    return attr->cloneAllAnnotations(ret);
```

Comment:
This is the fix.  See ecl/regress/minherit1.ecl for a simple example that is affected.  The new xml information is more correct.